# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openexr/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- OpenEXR-1.2.2-orig/exrmaketiled/Image.h	2004-06-04 04:11:23.000000000 +0000
+++ OpenEXR-1.2.2/exrmaketiled/Image.h	2005-08-23 05:13:56.000000000 +0000
@@ -49,6 +49,7 @@
 #include <string>
 #include <map>
 
+class Image;
 
 class ImageChannel
 {

